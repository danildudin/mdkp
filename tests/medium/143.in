28 10
56 390 63 555 736 941 185 565 76 416 192 775 75 764 696 895 130 311 340 804 638 194 736 981 844 279 170 150 
395 204 162 759 713 45 238 86 377 765 65 459 300 184 267 391 107 495 955 430 669 986 267 68 288 39 297 784 6734
295 42 840 684 333 784 709 778 887 235 52 100 879 467 558 403 400 675 474 248 628 200 306 670 741 374 338 720 9281
454 950 606 996 638 751 198 957 925 365 667 809 790 456 978 301 505 953 253 654 927 835 761 660 508 352 704 109 7870
928 4 85 106 170 617 137 375 395 61 225 772 555 966 967 903 300 27 729 905 938 432 281 550 589 724 272 660 5860
345 491 218 902 392 996 944 32 793 744 839 669 48 122 20 37 720 412 346 401 326 697 225 356 306 143 983 119 6853
316 376 998 392 103 78 613 265 550 427 729 427 958 951 786 870 390 774 521 281 872 781 147 132 116 139 188 437 9593
862 87 663 591 710 909 811 547 879 758 756 721 35 514 23 194 460 836 101 960 481 758 896 111 51 244 796 496 7880
429 10 963 174 1000 694 293 270 206 171 388 544 792 331 785 49 233 39 651 176 922 808 729 227 883 415 985 958 6079
372 386 57 508 429 238 858 837 956 946 820 708 22 996 79 815 586 416 444 166 157 590 840 935 945 288 684 424 5436
508 947 406 568 715 579 30 941 642 761 551 57 22 409 922 585 881 990 969 481 914 182 572 688 599 716 222 380 6934
