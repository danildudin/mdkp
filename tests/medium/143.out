7680
11
1 2 3 4 5 11 14 15 19 23 24 
